src/main/java/pageclasses/LandingPage.java

```diff
@@ -15,12 +17,12 @@ public void openLandingPage() {
     }
 
     public boolean isSignInDisplayed() {
-        return driver.findElement(MAIL_LOGIN).isDisplayed();
+        return mailLogin.isDisplayed();
     }
 
     public LoginPage clickSignIn() {
         // click button
```

Comment:
there is no need to add comments everywhere. Especially where everything is already obvious

Reply:
Sorry, forgot to remove those. They were placeholders for what I wanted to implement there and forgot about them